### ISSUE TITLE
Commander des stores pour la salle de conférence

### ISSUE THREAD

**Issue by lenaMK** [ouvroir/labouvroir#127]
essayer de trouver les mêmes

**Comment by lenaMK** [comment]
à part si on veut jouer sur les couleurs pour unifier bureau et salle de conf

**Comment by ZoeRenaudie** [comment]
commande lancée sur Maximo

**Comment by ZoeRenaudie** [comment]
Total de 1204$ pour 5 stores.. Voir mail Simon Truteau

**Comment by ZoeRenaudie** [comment]
Je leur envoie le devis?

**Comment by emchateau** [comment]
Oui, cela peut être utile. Mais je pense qu’il faut parler avant à Katrina pour qu’elle soit au courant de la situation. J'ai prévenu Michael.

**Comment by ZoeRenaudie** [comment]
Devis transféré au Crihn.
